### PR TITLE
feat(dataroom): update LinksWrapper

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves
- Passes a 'light' or 'dark' variant the component to be displayed in different backgrounds
- Update the Dune links according to the thread: https://5afe.slack.com/archives/C073BB9AS4E/p1720771881183709?thread_ts=1720424933.851669&cid=C073BB9AS4E
- Pre populate the social links with the Dune asset

### CODE REVIEW

File: src/components/Pressroom/FeaturedVideos/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const VideoCard = ({ title, url }: { title?: string; url: string }) => (
   <div className={css.card}>
     <MediaPlayer url={url} />
     <div className={css.cardBody}>
-      {title ? <Typography variant="h3">{title}</Typography> : null}
+      {title ? <Typography variant="h4">{title}</Typography> : null}
```
Comment (DiogoSoaress) [inline src/components/Pressroom/FeaturedVideos/index.tsx:13]: This is an unrelated typography adjustment I accidentally commit.

**Review by iamacook — APPROVED** [review]
